TINY-4589: Rtl/ltl attributes sometimes gets applied to list items (#6972)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Changed the load order so that the content css gets loaded before the editor gets populated with contents. #TINY-7249

### Fixed

> - Rtl/ltl attributes sometimes gets applied to list items #TINY-4589

Reviewer: Maybe

```suggestion
- The `dir` attribute was being incorrectly applied to list items #TINY-4589
```
